### ISSUE TITLE
[Bug]: update 1.9.0 stable diffusion stops when using hires fix in tiled upscaler 12 steps

### ISSUE THREAD

**Issue by Dshinhal** [lshqqytiger/stable-diffusion-webui-directml#443]
### Checklist

- [ ] The issue exists after disabling all extensions
- [ ] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

I can't use hires fix in this latest update it doesn't go beyond tiled upscaler

### Steps to reproduce the problem

I don't know

### What should have happened?

I come from version 1.8.0/1.8.0 RC and it worked in 1.9.0 RC and the latest 1.9.0 no longer works it does not pass the hires process it stays stuck in tiled upscaler

### Sysinfo

[sysinfo-2024-04-16-05-09.json](https://github.com/lshqqytiger/stable-diffusion-webui-directml/files/14987056/sysinfo-2024-04-16-05-09.json)

### Console logs

```Shell
To create a public link, set `share=True` in `launch()`.
Startup time: 14.1s (prepare environment: 19.1s, initialize shared: 3.3s, list SD models: 0.2s, load scripts: 3.4s, create ui: 0.6s, gradio launch: 0.5s).
Loading VAE weights specified in settings: C:\Users\daxtr\Downloads\AUTOMATICA 1111\stable-diffusion-webui-directml\models\VAE\
Applying attention optimization: sub-quadratic... done.
Model loaded in 15.7s (load weights from disk: 1.0s, create model: 0.4s, apply weights to model: 11.6s, apply float(): 0.8s, load VAE: 1.3s, calculate empty prompt: 0.5s).
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [01:53<00:00,  5.66s/it]
tiled upscale: 100%|███████████████████████████████████████████████████████████████████| 12/12 [00:18<00:00,  1.50s/it]
```

### Additional information

I installed everything again and nothing. sometimes it generates an image without problems but the next one gets stuck in tiled upscale

**Comment by neko47834** [comment]
![Captura de pantalla (1)](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/40369442/f2053793-471d-412a-9e0d-2a31d1686d49)

same thing...upgraded to version 1.9.0 and now it gets stuck when trying to use Hires.fix

**Comment by Dshinhal** [comment]
> ![Captura de pantalla (1)](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/40369442/f2053793-471d-412a-9e0d-2a31d1686d49)
> 
> same thing...upgraded to version 1.9.0 and now it gets stuck when trying to use Hires.fix

I went back to version 1.8.0-RC and it was fixed, waiting for an update to 1.9.0

**Comment by lshqqytiger** [comment]
What gpu are you using?
